pkgs/development/python-modules/aiohttp-socks/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchPypi, buildPythonPackage, pythonOlder, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "aiohttp-socks";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "aiohttp_socks";
+    sha256 = "0473702jk66xrgpm28wbdgpnak4v0dh2qmdjw7ky7hf3lwwqkggf";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
+
+  # Checks needs internet access
```

Comment:
I don't think so, but they use localhost, to which access is denied in the sandbox.

Reply:
Isn't this a remote address ? https://github.com/romis2012/aiohttp-socks/blob/3252f4bdd37fb9a7360481977f800189cb3e3aca/tests/test_socks.py#L18

Reply:
Indeed, it is. Sorry.